test/js-native-api/js-native-api.status

```diff
@@ -0,0 +1,25 @@
+prefix js-native-api
+
+# To mark a test as flaky, list the test name in the appropriate section
+# below, without ".js", followed by ": PASS,FLAKY". Example:
+# sample-test                        : PASS,FLAKY
+
+[true] # This section applies to all platforms
+
+[$system==win32]
+
+[$system==linux]
+
+[$system==macos]
+
+[$arch==arm || $arch==arm64]
+
+[$system==solaris] # Also applies to SmartOS
+
+[$system==freebsd]
+
+[$system==aix]
+
+[$type==coverage]
+test_function/test: PASS,FAIL,CRASH
```

Comment:
I like this approach to isolating flaky tests a lot 👍 first saw it in the blink codebase.

Reply:
You might consider putting this in https://github.com/nodejs/node/blob/master/test/root.status
My intuition is that file fit better to represent this is a cross-cutting concern.

Reply:
You learn something new every day.  I was not aware of that option.  I like having the list in one place so will move there.